docs/best-practices/code-reviews/readme.md

```diff
@@ -1,129 +1,154 @@
+[shorter guides](https://github.com/thoughtbot/guides/tree/master/code-review),
+[longer checklists](https://www.codeproject.com/Articles/524235/Codeplusreviewplusguidelines),
+[humanized reviews](http://www.processimpact.com/articles/humanizing_reviews.html),
+[reasons for doing code reviews in the first place](https://smartbear.com/learn/code-review/why-review-code/),
+[best practices](https://smartbear.com/learn/code-review/best-practices-for-peer-code-review/),
+[more best practices](https://www.atlassian.com/agile/software-development/code-reviews),
+[statistics on code review effectiveness for catching bugs](https://blog.codinghorror.com/code-reviews-just-do-it/),
+and [examples of code reviews gone wrong](https://blog.fogcreek.com/effective-code-reviews-9-tips-from-a-converted-skeptic/).
+Oh, and of course there is are
+[books](https://books.google.com/books/about/Peer_Reviews_in_Software.html?id=d7BQAAAAMAAJ),
+too.  Long story short, this document presents Palantir's take on code reviews.
+Organizations with deep cultural reluctance to peer reviews may want to consult
+Karl E. Wiegers' excellent essay on [Humanizing Peer Reviews](http://www.processimpact.com/articles/humanizing_reviews.html) before trying to follow this guide.
 
 This guide covers the following topics:
 
-- [What to Review](#what-to-review)
-- [Preparing Code for Review](#preparing-code-for-review)
-- [Writing Useful Commit Messages](#writing-useful-commit-messages)
-- [Doing Code Reviews](#doing-code-reviews)
-  - [Purpose](#purpose)
-  - [Implementation](#implementation)
-  - [Legibility & Style](#legibility-style)
-  - [Maintainability](#maintainability)
-  - [Review Comments: Concise and Actionable](#review-comments-concise-and-actionable)
-- [Responding to Reviews](#responding-to-reviews)
-- [Code Review Examples](#code-review-examples)
-  - [Example: Inconsistent Naming](#example-inconsistent-naming)
-  - [Example: Inconsistent Method Signatures](#example-inconsistent-method-signatures)
-  - [Example: Code Style Violation](#example-code-style-violation)
-  - [Example: Library Use](#example-library-use)
-  - [Example: Potential Bug](#example-potential-bug)
-
-We do code reviews (CRs) to **ensure high code quality** and to **reduce the
-number of bugs** we ship. High-quality code is **clean**, **correct**,
-and **scalable**. In more detail, reviews serve several important
-causes:
-
-- **Committers are motivated** subconsciously by the notion of a set
+- [Motivation](#motivation)
+- [What to review](#what-to-review)
+- [Preparing code for review](#preparing-code-for-review)
+- [Commit messages](#commit-messages)
+- [Performing code reviews](#performing-code-reviews)
+- [Responding to reviews](#responding-to-reviews)
+- [Examples](#examples)
+
+
+## Motivation
+
+Code reviews (CRs) improve code quality and have positive effects on team and company culture. In more detail:
```

Comment:
I’d be tempted to add a qualifier after (CRs), such as “generally” or “will almost surely”.

Reply:
done